### PR TITLE
EDM-695: Logger verbosity flag was added (-v)

### PR DESCRIPTION
**Pull request by gshilin-sdb**
Logger verbosity level may be supplied via -v flag

It may be one of "fatal", "error", "warn", "warning", "info", "debug"

Default: "debug"

### CODE REVIEW

**Comment by gshilin-sdb** [comment]
cc @yobshans

File: cmd/devicesimulator/main.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/flightctl/flightctl/internal/util"
 	flightlog "github.com/flightctl/flightctl/pkg/log"
 	testutil "github.com/flightctl/flightctl/test/util"
-	"github.com/sirupsen/logrus"
+	logrus "github.com/sirupsen/logrus"
```
[the remark this answers is not in the document]
Comment (gshilin-sdb) [reply cmd/devicesimulator/main.go:24]: oops

File: cmd/devicesimulator/main.go [hunk 2]
```diff
@@ -55,6 +56,12 @@ func main() {
 	}
 	pflag.Parse()
 
+	logLvl, err := logrus.ParseLevel(*logLevel)
+	if err != nil {
+		logLvl = logrus.DebugLevel
```
[the remark this answers is not in the document]
Comment (gshilin-sdb) [reply cmd/devicesimulator/main.go:61]: DebugLevel means that any level equal or higher than Debug (i.e. all but Trace, and it is not used anywhere) will be reported. This what happens now, so I thought it'll be good as a default.

[the remark this answers is not in the document]
Comment (gshilin-sdb) [reply cmd/devicesimulator/main.go]: updated

**Review by hexfusion — APPROVED** [review]

File: cmd/devicesimulator/main.go [hunk 3]
```diff
@@ -47,6 +47,7 @@ func main() {
 	initialDeviceIndex := pflag.Int("initial-device-index", 0, "starting index for device name suffix, (e.g., device-0000 for 0, device-0200 for 200))")
 	metricsAddr := pflag.String("metrics", "localhost:9093", "address for the metrics endpoint")
 	stopAfter := pflag.Duration("stop-after", 0, "stop the simulator after the specified duration")
+	logLevel := pflag.StringP("v", "v", "debug", "logger verbosity level (one of \"fatal\", \"error\", \"warn\", \"warning\", \"info\", \"debug\")")
```
Comment (hexfusion) [inline cmd/devicesimulator/main.go:50]: If you want to use StringP which gives you a long a and a short flag should we make those different?
```suggestion
	logLevel := pflag.StringP("log-level", "v", "debug", "logger verbosity level (one of \"fatal\", \"error\", \"warn\", \"warning\", \"info\", \"debug\")")
```

Comment (gshilin-sdb) [reply cmd/devicesimulator/main.go:50]: just wanted to give an option to supply `--v` and `-v`... but your version is also quite good. changing

**Review by hexfusion — APPROVED** [review]

**Pull request merged**